src/cli/onefuzz/backend.py

```diff
@@ -143,7 +145,8 @@ def load_config(self) -> None:
     def save_config(self) -> None:
```

Comment:
when does this get called ?

Reply:
When a cli user calls `onefuzz config --endpoint ...`